cmd/kms-rest/go.sum

```diff
@@ -3,26 +3,38 @@ cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
```

Comment:
Running `go mod tidy` might help clean up this file and make the PR a bit shorter

Reply:
it was done, I am trying to do that whenever changes are done to go.mod